content/docs/command-reference/remote/modify.md

```diff
@@ -25,16 +27,38 @@ positional arguments:
 
 ## Description
 
-Remote `name` and `option` name are required. Config option names are specific
-to the remote type. See `dvc remote add` and
-[Available parameters](#available-parameters-per-storage-type) below for a list
-of remote storage types.
+The DVC remote's `name` and a valid `option` to modify are required. Remote
+options or [config parameters](#available-parameters-per-storage-type) are
+specific to the storage type and typically require a `value` as well.
```

Comment:
> disambiguate vs. the `option` argument (remote/config parameters)

Should we instead change DVC to call the `option` argument something else? I'm not sure I love the term config param either though, since we have `dvc params`, ML hyperparams, etc... Cc @dberenbaum

Reply:
> I'm not sure I love the term config param either though

Yeah, TBH I don't see `option` as particularly problematic and the alternatives don't seem obviously better, so I think it's fine to leave it.

Reply:
OK but maybe then we should avoid "param" ? Or is it fine to keep both

E.g. [Available parameters for all remotes](https://dvc.org/doc/command-reference/remote/modify#available-parameters-for-all-remotes)

Reply:
IMO it's not that confusing in this case and I wouldn't bother changing it unless someone has been confused by it.